index.html

```diff
@@ -132,5 +123,251 @@
     
         </section>
     </header>
+
+    <main class="works">
+        <div class="card-works">
+                <div class="work-snapshot">
+                </div>
+                <div class="work-block">
+                    <div class="primary-text">
+                        <h1 class="project-title">Tonic</h1>
+                        <div class="frame">
+                            <div class="client"><p>CANOPY</p></div>
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="role"><p>Back&nbspEnd&nbspDev</p></div>
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="year"><p>2015</p></div>
+                        </div>
+                    </div>
+                    <h2 class="project-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+                    <ul class="tags">
+                        <li class="tags-list"><a class="tags-text" href="">html</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">css</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">javaScript</a></li>
+                    </ul>
+                    <div class="actions">
+                        <button>
+                            <a href="">See&nbspProject</a>
+                        </button>
```

Comment:
- [ Required ] The See project button can be either a `button` or `a` tag but using both is not good practice, I would like to suggest that you use the `button` tag. Please consider changing this in all the `See project` buttons in the works section.